### ISSUE TITLE
[SDK Automation] Add a Check to verify SDK automation configuration in readme.md or tspConfig.yaml in azure-rest-api-specs CI pipeline

### ISSUE THREAD

**Issue by chunyu3** [Azure/azure-sdk-tools#9291]
azure-rest-api-spec pipeline framework identify if it need trigger SDK automation for language according the SDK automation configuration in `readme.md` for swagger project, and `tspConfig.yaml` for typespec project.

But now some typespec project also add SDK automation in `readme.md` which cause pipeline framework ask to trigger SDK automation from swagger which cause the SDK automation failure.
see
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4270662&view=logs&j=03afb3bb-7296-55ad-aa07-ceee610c73b2&t=0f320c29-1c90-5304-d2d3-c58a6bb76437&s=eb9754a7-3885-5b5b-bd91-16c95dd7881e

Add a SDK automation configuration check for azure-rest-api-specs PR CI pipeline:

- when it is typespec project, the SDK automation configuration only exist in `tspConfig.yaml`. `readme.md` should not contain this.
- When it is swagger project, the SDK automation configuration can defined in `readme.md`

**Comment by kurtzeborn** [comment]
Possibly a duplicate of: https://github.com/Azure/azure-sdk-tools/issues/9355

**Comment by raych1** [comment]
Yes, this is similar. @wanlwanl could you see if this can be covered when you work on the #9355 ?

**Comment by wanlwanl** [comment]
Yes, this is duplicated, close this one.
